sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -3,68 +3,39 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_chiller01P01_enable</EFDB_Topic>
-    <Description>Enable command for the chiller01P01 device.</Description>
+    <EFDB_Topic>HVAC_command_enable_device</EFDB_Topic>
+    <Description>Enable command for a single HVAC device.</Description>
```

Comment:
Please consider "Enable a single HVAC device" (active voice).

And same for the disable command.

Reply:
Yes I keep forgetting about that. Done.